### PR TITLE
Fix capitalization of audioRoutingMixing package

### PR DESCRIPTION
**Pull request by palemieux**

### CODE REVIEW

File: src/main/java/com/netflix/imflibrary/st2067_100/OutputProfileListModel_st2067_100_2014.java [hunk 1]
```diff
@@ -226,22 +226,22 @@ private PixelEncoderMacro createPixelEncoderMacro(PixelEncoderType pixelEncoderT
                 input, output);
     }
 
-    private AudioRoutingMixingMacro createAudioRoutingMixingMacro(AudioRoutingMixingMacroType audioRoutingMixingMacroType) {
-        if(audioRoutingMixingMacroType.getOutputEntityList() == null) {
+    private AudioRoutingMixingMacro createAudioRoutingMixingMacro(AudioRoutingMixingMacroType AudioRoutingMixingMacroType) {
```
Comment (fschleich) [inline src/main/java/com/netflix/imflibrary/st2067_100/OutputProfileListModel_st2067_100_2014.java:229]: @palemieux - I don't follow why the parameter names should be capitalized, can you explain?

Comment (palemieux) [reply src/main/java/com/netflix/imflibrary/st2067_100/OutputProfileListModel_st2067_100_2014.java:229]: That was a mistake resulting from a search and replace that was too zealous.

Fixed at https://github.com/sandflow/photon/tree/issues/correct-capitalization

Comment (fschleich) [reply src/main/java/com/netflix/imflibrary/st2067_100/OutputProfileListModel_st2067_100_2014.java]: thanks!

**Review by fschleich — APPROVED** [review]
lgtm

**Pull request merged**